pandas/util/testing.py

```diff
@@ -2542,9 +2543,8 @@ def assert_produces_warning(expected_warning=Warning, filter_level="always",
                                     % extra_warnings)
 
 
-def disabled(t):
-    t.disabled = True
-    return t
+disabled = pytest.mark.disabled
```

Comment:
I don't think we actually have any disabled tests, so fine to remove this marker

but I think you need to make ``network`` a market as well